src/test/java/core/basesyntax/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    private static WriterService writerService;
+
+    @BeforeAll
+    static void setup() {
+        writerService = new WriterServiceImpl();
+    }
+
+    @Test
+    void writeToFile_successful() {
+        List<String> testList = List.of("apple,10", "banana,10");
+        String address = "testReport.csv";
+        try {
+            List<String> testFile = Files.readAllLines(Path.of(address));
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
we don't need to use try catch in unit tests, if the exception is thrown - test is failed automatically, if we expect it - we use assertThrows